specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -7856,19 +7959,15 @@
       ],
       "description": "The Setting.",
       "discriminator": "kind",
-      "type": "object",
-      "required": [
-        "kind"
-      ]
+      "type": "object"
     },
     "SettingsKind": {
       "description": "Describes an Azure resource with kind.",
       "properties": {
         "kind": {
           "description": "The kind of the setting",
           "enum": [
-            "UebaSettings",
-            "ToggleSettings"
+            "EyesOn"
```

Comment:
As per Swagger update guidance this is breaking change. Please confirm if this is not being used and share the details.

Reply:
@chiragg4u 
For breaking change, here is author's explanation: https://github.com/Azure/azure-rest-api-specs/pull/9355#issuecomment-625095704

Reply:
@chiragg4u 
The "Settings" endpoint was a draft and never being used(I verified that)
We now implemented this endpoint with the first kind "eyesOn"

verified that there is no use on this endpoint for the last 60 days

Reply:
@chiragg4u fixed conflicts